plugins/TimingTriggerCandidateMaker.hpp

```diff
@@ -47,7 +50,7 @@ class TimingTriggerCandidateMaker : public dunedaq::appfwk::DAQModule
 
   std::chrono::milliseconds queueTimeout_;
 
-  std::vector<std::pair<int64_t, int64_t>> m_map;
+  std::vector<std::pair<int64_t, int64_t>> m_detid_offsets_map;
```

Comment:
Now that I think about it, let's make this a `std::map`. That more clearly represents what it's doing, and it means we don't need to have offsets for all values starting at zero (eg, we could store offsets for just 1, 3 and 5)

Reply:
Ok